### DISCUSSION TITLE
Can the Home-Assistant Integration pull the watch's "last known location" from Xplora (without polling the watch)?

### DISCUSSION THREAD

**Discussion by mwheler** [Ludy87/xplora_watch#445] [Q&A]
Good day,
First off, thanks very much for this integration, I have it installed and working for a couple of Xplora X5 Play's using the experimental email login without creating an additional user account (and the app is working on my phone without being logged out).

I'm hoping there is a way to poll Xplora API from the HA Integration on a short interval (60-300 seconds) to get the "last known location" without polling the watch directly which seems to drain the watch battery.  It looks like it is maybe an option on the pyxlpora_api page, but I don't see it on the HA integration.

Ideally I'd like HA to update the location as soon as Xplora has captured a new location. Based on the history I see in the Xplora App, I think this would be frequent enough for my use.

If this is already possible, please help me understand how to use it.  If not, is this something that can be added (I'm not a developer...)?

Thank you!

**Comment by Ludy87** [comment]
Hi, every location query has to trigger the watch, which is why the battery is drained. The watch does not send locations, but always queries live.
